docs/models/BERT/input_variations.md

```diff
@@ -6,7 +6,7 @@
 |  2 | aten.add.Tensor                |                  2 |           2 |
 |  3 | aten.addmm.default             |                  4 |           4 |
 |  4 | aten.bmm.default               |                  2 |           2 |
-|  5 | aten.clone.default             |                  4 |           4 |
+|  5 | aten.clone.default             |                  4 |           2 |
```

Comment:
There is a regression in the reporting of some ops that are wrapped https://github.com/tenstorrent/pytorch2.0_ttnn/blob/30d67d08f047734297b37c9c4cd989057ba50fbb/torch_ttnn/passes/lowering/target_wrappers.py#L5-L7

These ops are actually not correctly recorded, so some new lowerings like the `aten.view` to `ttnn.reshape` can expose this problem. I'll will fix this in the next PR.